src/ptf/testutils.py

```diff
@@ -1858,6 +1858,35 @@ def verify_packets_any(test, pkt, ports=[], device_number=0):
 
     test.assertTrue(received == True, "Did not receive expected pkt on any of ports %r for device %d" % (ports, device_number))
 
+def verify_packet_any_port(test, pkt, ports=[], device_number=0):
```

Comment:
how is it different from `verify_any_packet_any_port` below, called with a single packet in the `pkts` list?